bears/r/FormatRBear.py

```diff
@@ -66,15 +65,18 @@ def create_arguments(filename, file, config_file,
                  'comment={r_keep_comments},'
                  'blank={r_keep_blank_lines},'
                  'arrow={r_use_arrows},'
-                 'width.cutoff={r_max_expression_length},'
                  'brace.newline={r_braces_on_next_line},'
-                 'indent={tab_width}'
-                 ')'.format(tab_width=tab_width,
-                            r_keep_comments=_map_to_r_bool(r_keep_comments),
-                            r_keep_blank_lines=_map_to_r_bool(
-                                r_keep_blank_lines),
-                            r_use_arrows=_map_to_r_bool(r_use_arrows),
-                            r_braces_on_next_line=_map_to_r_bool(
-                                r_braces_on_next_line),
-                            r_max_expression_length=r_max_expression_length))
+                 'indent='
+                 '{tab_width}'.format(tab_width=tab_width,
+                                      r_keep_comments=_map_to_r_bool(
+                                          r_keep_comments),
+                                      r_keep_blank_lines=_map_to_r_bool(
+                                          r_keep_blank_lines),
+                                      r_use_arrows=_map_to_r_bool(
+                                          r_use_arrows),
+                                      r_braces_on_next_line=_map_to_r_bool(
+                                          r_braces_on_next_line)))
+        # Disable r_max_expression_length if it is equal to 0
+        rcode += ('' if r_max_expression_length ==
+                  0 else ',width.cutoff=' + str(r_max_expression_length)) + ')'
         return '-e', rcode
```

Comment:
commit message shortlog: I like the title, though I think you should replace "Enable" with "Allow" and "disablement" with "disabling" (you should look into a dictionary for "disablement" :P)